content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+code. For special cases where we discover a proprietary code that does
+not map cleanly to any standard, we will work with you to figure out the
+best option for that mapping during deployment.
+
+For each of these scenarios, it is one thing to validate against our
+sandbox to adhere to the FHIR specification, but we also need to
+validate your app at each client site to ensure that there aren’t any
+hospital-specific considerations to be aware of. We will do this when we
+connect your app to a hospital domain. We will work with you to resolve
+any of these hospital-specific considerations.
+
+The above examples use lead, but similar considerations could exist for
+virtually any lab or measurement where concept mapping and specific
+LOINC codes are used.  If your app uses specific LOINC codes to query
+for information,
+[contact us](https://groups.google.com/d/forum/cerner-fhir-developers)
+and we will help you work through any considerations that might exist.
+
+### Leverage Query Parameters
+
+We have seen a lot of instances of developers performing post-query
+filtering rather than leveraging the
+[query parameters](https://www.hl7.org/fhir/dstu2/search.html#table)
+defined by the API.  We recommend using the available query parameters, such as
+[this example for the Patient resource](http://fhir.cerner.com/millennium/dstu2/individuals/patient/#parameters).
+Query parameters make the dataset more predictable, speed up your searches, and
+are more reliable than relying on post-query filtering.
+
+### Review Filtering Options
+
+Using Query Parameters is what we recommend instead of post-query filtering,
+but if you are filtering, here is one consideration to be aware of. In the
+Cerner EMR, if a medication or result is documented erroneously in the
+system and then is deleted by a user, it is commonly represented in the
+system as In Error. In Error results are frequently filtered out and are
+not displayed by native Cerner apps.
+
+You should review your app’s status filtering logic to ensure that the
+appropriate results are displayed. For Cerner’s implementation of the
+HL7 FHIR standard (based on DSTU 2 Final (1.0.2)), it is particularly
+recommended for the [MedicationOrder](http://fhir.cerner.com/millennium/dstu2/medications/medication-order/)
+or [Observation](http://fhir.cerner.com/millennium/dstu2/diagnostic/observation/)
+resources. It would be a patient safety issue if an uncharted value was
+displayed to a user. Please double-check the status codes you use from
+the FHIR [MedicationOrderStatus](http://hl7.org/fhir/DSTU2/valueset-medication-order-status.html)
+or [ObservationStatus](https://www.hl7.org/fhir/dstu2/valueset-observation-status.html)
+value sets.
+
+Typically, for MedicationOrder, only medications in `Active`, `On-Hold`, or
+`Completed` statuses are displayed.  For Observation, only results in
+`Final` and `Amended` statuses are displayed.
+
+In our sandbox domain, Amber Read, PATIENT_ID 4614008, has some entered-
```

Comment:
whatever we decide to do with good 'ole Timmy would follow here. If we go this route, instead of PATIENT_ID, perhaps Patient.id.